### ISSUE TITLE
BedAura (+1.13.2 mode)

### ISSUE THREAD

**Issue by Ev3yr** [kami-blue/client#1232]
Bedaura - acts like crystalaura but for beds. Automatically places beds in the position that will deal the most damage to the player targetted. Not really that useful in 1.12.2 because of crystals BUT in 1.13.2 beds were made able to be placed on single blocks and can float, meaning that it's possible to place beds on top of players who would otherwise have been safe due to obsidian at their feet and deal massive explosion damage. Beds can also just straight-up be placed in the air, though that's probably some weird finnicky packet stuff.

Basically, bedaura auto places beds and explodes them in the nether. 1.13+ mode will allow for new placing that can place inside players when they would be safe from crystals and would also place in the air if necessary, eg against people using elytra. Reason for the 1.13 mode on a 1.12.2 client is because some servers allow a range of game versions to join, like endcrystal.me.
